plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/ui/AccountSelectorObservableValue.java

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+package com.google.cloud.tools.eclipse.appengine.login.ui;
+
+import org.eclipse.core.databinding.observable.Diffs;
+import org.eclipse.core.databinding.observable.value.AbstractObservableValue;
+import org.eclipse.swt.events.SelectionAdapter;
+import org.eclipse.swt.events.SelectionEvent;
+
+/**
+ * An object that represents and binds to the email value of an {@link AccountSelector}, whose
+ * changes (i.e., account selection changes in the combo box) can be tracked by value change
+ * listeners.
+ */
+public class AccountSelectorObservableValue extends AbstractObservableValue {
+
+  private String oldValue;
+  private AccountSelector accountSelector;
+
+  public AccountSelectorObservableValue(final AccountSelector accountSelector) {
```

Comment:
Because this is an SWT-specific value, you need to set the realm.  Something like the following should work:
```
super(DisplayRealm.forDisplay(accountSelector.getDisplay()));
```
(The default `Realm` may well be a `DisplayRealm`, but better to be safe than sorry.)